airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py

```diff
@@ -132,10 +132,9 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         Show error message in case of request exception or unexpected response.
         """
         try:
-            config = AmazonSellerPartnerConfig.parse_obj(config)  # FIXME: this will be not need after we fix CDK
```

Comment:
Could you please be more specific with your question?

Reply:
What is the `old config` ? 

if I understand you correctly, this is the input *.json config file? If so, the answer is yes, because the old config still requires the `app_id` that is no longer present, but was required. So this is the breaking change in either way.